Docker/Production/entando-full-stack/microengine/hooks/post_push

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+LATEST_VERSION="5.0.1-SNAPSHOT"
```

Comment:
In a lot of the other files we take this as an arg in ENTANDO_VERSION_TO_BUILD.  Should this be an arg as well?

Reply:
The two variables have different semantics associated with them. The ENTANDO_VERSION_TO_BUILD could be any version you may want to build (5.0.1-SNAPSHOT, 5.1.0-SNAPSHOT, 5.0.1), LATEST_VERSION is specifically the version that we tag as 'latest' in Docker. Personally I like to know exactly which version to use so I never use 'latest' However the 'latest' tag is a Docker standard so we implement it to be polite to the Docker community. Once we release 5.0.1, I will do a global search and replace in the git repo to change 5.0.1-SNAPSHOT to 5.0.1